samples/python/hosts/cli/pyproject.toml

```diff
@@ -7,6 +7,7 @@ requires-python = ">=3.12"
 dependencies = [
     "a2a-samples",
```

Comment:
Can you confirm that this CLI is using the A2A SDK and not the files from common?